js-intro.js

```diff
@@ -1,33 +1,58 @@
-// ASSESSMENT 1: INTRO TO JAVASCRIPT
-// Coding practical questions
+// // ASSESSMENT 1: INTRO TO JAVASCRIPT
+// // Coding practical questions
 
-// Please read all questions thoroughly
-// Psuedo coding is highly recommended
+// // Please read all questions thoroughly
+// // Psuedo coding is highly recommended
 
-// --------------------1) Create a function that determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Farhenheit.
-// Use the test variables provided below. Expected outcome: "35 is below boiling point", "350 is above boiling point", "212 is at boiling point"
+// // --------------------1) Create a function that determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Farhenheit.
+// // Use the test variables provided below. Expected outcome: "35 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
 var temp1 = 35
 var temp2 = 350
 var temp3 = 212
 
+// //create a function called checkBoilingPoint
+ checkBoilingPoint = (temp) => {
+// function checkBoilingPoint (temperature){
+    if(temp === 212){
+        return temp + " is at boiling point"
+    }else if(temp < 212){
+        return temp + " is below boiling point"
+    }else {
+        return temp + " is above boiling point"
+    }
+}
```

Comment:
Great job with this logic! I would consider using an `else if` instead of an `else` statement, just in case you want to include the opportunity for errors in your user input in the future :) This code still totally works, just a suggestion!